addons/hr/hr.py

```diff
@@ -141,16 +141,16 @@ class hr_employee(osv.osv):
         #we need a related field in order to be able to sort the employee by name
         'name_related': fields.related('resource_id', 'name', type='char', string='Name', readonly=True, store=True),
         'country_id': fields.many2one('res.country', 'Nationality (Country)'),
-        'birthday': fields.date("Date of Birth"),
-        'ssnid': fields.char('SSN No', help='Social Security Number'),
-        'sinid': fields.char('SIN No', help="Social Insurance Number"),
-        'identification_id': fields.char('Identification No'),
-        'gender': fields.selection([('male', 'Male'), ('female', 'Female'), ('other', 'Other')], 'Gender'),
-        'marital': fields.selection([('single', 'Single'), ('married', 'Married'), ('widower', 'Widower'), ('divorced', 'Divorced')], 'Marital Status'),
+        'birthday': fields.date("Date of Birth", groups="base.group_hr_user"),
+        'ssnid': fields.char('SSN No', help='Social Security Number', groups="base.group_hr_user"),
+        'sinid': fields.char('SIN No', help="Social Insurance Number", groups="base.group_hr_user"),
+        'identification_id': fields.char('Identification No', groups="base.group_hr_user"),
+        'gender': fields.selection([('male', 'Male'), ('female', 'Female'), ('other', 'Other')], 'Gender', groups="base.group_hr_user"),
+        'marital': fields.selection([('single', 'Single'), ('married', 'Married'), ('widower', 'Widower'), ('divorced', 'Divorced')], 'Marital Status', groups="base.group_hr_user"),
         'department_id': fields.many2one('hr.department', 'Department'),
         'address_id': fields.many2one('res.partner', 'Working Address'),
         'address_home_id': fields.many2one('res.partner', 'Home Address'),
```

Comment:
They are, but since the address info is stored as a regular `res.partner` entry, all employees have read access by default anyway, they only have to search. So instead we'd need a security rule that selectively blocks access to those kinds of addresses, which probably implies changing the model to mark them as "private/home addresses". This part of the fix would likely have to land in master...  so I'd prefer to leave it out for later and still land the current patch in stable, as it should be *relatively* safe.